includes/core/classes/class-event.php

```diff
@@ -399,6 +399,14 @@ protected function get_formatted_datetime(
 		string $which = 'start',
 		bool $local = true
 	): string {
+
+		$cache_key = 'formatted_datetime_' . md5($format . $which . ($local ? 'local' : 'gmt'));
+
```

Comment:
I would remove this line break and line up `=` also add more whitespace to meet WordPress standards.
```
$cache_key   = 'formatted_datetime_' . md5( $format . $which . ( $local ? 'local' : 'gmt' ) );
$cached_date = get_transient( $cache_key );
```